src/plugins/intel_npu/src/backend/include/zero_pipeline.hpp

```diff
@@ -21,13 +21,15 @@ struct TensorData {
 struct Pipeline {
 public:
     Pipeline(const Config& config,
-             const std::shared_ptr<const IExecutor>& executorPtr,
+             const std::shared_ptr<ZeroInitStructsHolder>& initStructs,
+             const std::shared_ptr<IGraph>& graph,
              zeroProfiling::ProfilingPool& profiling_pool,
              zeroProfiling::ProfilingQuery& profiling_query,
              std::shared_ptr<zeroProfiling::NpuInferProfiling> npu_profiling,
              const std::vector<std::vector<std::optional<TensorData>>>& inputTensorsData,
              const std::vector<std::optional<TensorData>>& outputTensorsData,
-             const size_t numberOfCommandLists);
+             const size_t numberOfCommandLists,
```

Comment:
Nitpick:
```suggestion
             size_t numberOfCommandLists,
```

`const` is not really needed here.

Reply:
done, thx